cli/src/main/scala/com/codacy/analysis/cli/command/analyse/AnalyseExecutor.scala

```diff
@@ -75,7 +88,9 @@ class AnalyseExecutor(toolInput: Option[String],
   private def analyseFiles(tool: Tool,
                            filesTarget: FilesTarget,
                            localConfigurationFile: Either[String, CodacyConfigurationFile]): Try[Set[ToolResult]] = {
-    val fileTarget = fileCollector.filter(tool, filesTarget, localConfigurationFile, remoteProjectConfiguration)
+    val analysisFilesTarget =
+      fileCollector.filter(tool, filesTarget, localConfigurationFile, remoteProjectConfiguration)
```

Comment:
We need to filter files here by the language we are running and not the languages the tool supports

Reply:
I think I want to do this in a different PR.